src/api-client.ts

```diff
@@ -0,0 +1,114 @@
+import * as httpm from "@actions/http-client";
+
+const userAgent = "GitHub multi-repository variant analysis";
+
+export function getApiClient() {
+  return new httpm.HttpClient(userAgent, [], {
+    allowRetries: true,
```

Comment:
Just curious, what's the default behaviour for this? Is it exponential retries?

Reply:
I didn't immediately find docs on this (maybe there are some) but it does look like exponential backoff: https://github.com/actions/toolkit/blob/main/packages/http-client/src/index.ts#L432